app/src/main/java/eu/kanade/tachiyomi/ui/more/OverflowDialog.kt

```diff
@@ -79,7 +79,8 @@ class OverflowDialog(activity: MainActivity) : Dialog(activity, R.style.Overflow
             dismiss()
         }
 
-        binding.aboutItem.text = context.getString(R.string.about).withSubtitle(binding.aboutItem.context, "v${BuildConfig.VERSION_NAME}")
+        val version = if (BuildConfig.DEBUG) "r" + BuildConfig.COMMIT_COUNT else "v" + BuildConfig.VERSION_NAME
+        binding.aboutItem.text = context.getString(R.string.about).withSubtitle(binding.aboutItem.context, version)
```

Comment:
dont bother with this. since debug isn't going to be shared out anyway

its also unrelated to this pr anyway